test/connection.test.js

```diff
@@ -885,7 +886,11 @@ describe('connections:', function() {
     return Model.create({ name: 'test' });
   });
 
-  it('throws a MongooseServerSelectionError on server selection timeout (gh-8451)', () => {
+  it('throws a MongooseServerSelectionError on server selection timeout (gh-8451)', function() {
+    if (typeof Deno !== 'undefined') {
+      // In Deno dns throws an uncatchable error here.
+      return this.skip();
+    }
```

Comment:
btw, are there any news regarding this "uncatchable dns error"?

Reply:
@hasezoey nope, I don't think this issue is significant enough to block shipping deno support.

Reply:
didnt mean to block with this question, just wanted to ask if there are any known updates regarding this problem to re-enable the test for deno